openfl/component/assigner/random_grouped_assigner.py

```diff
@@ -35,12 +39,12 @@ class RandomGroupedAssigner(Assigner):
         \* - Plan setting.
     """
 
-    def __init__(self, task_groups, **kwargs):
+    def __init__(self, task_groups: List[object], **kwargs) -> None:
```

Comment:
```suggestion
    def __init__(self, task_groups: List[Any], **kwargs) -> None:
```
Because of `Any` means the same as `object` in python?